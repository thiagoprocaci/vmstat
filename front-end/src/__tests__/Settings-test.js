'use strict';

import React from 'react';
import Settings from '../js/components/Settings.jsx';
import renderer from 'react-test-renderer';

describe('Settings', () => {

  it('renders correctly', () => {
    const tree = renderer.create(
      <Settings />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});