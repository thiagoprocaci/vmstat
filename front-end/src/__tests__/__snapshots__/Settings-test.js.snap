exports[`Settings renders correctly 1`] = `
<h1>
  Settings
</h1>
`;
